RxSwiftWorkshop/Maps/Model/GeolocatorSpec.swift

```diff
@@ -34,47 +33,89 @@ class GeolocatorSpec: QuickSpec {
                 sut = nil
             }
 
-            it("should pass geolocation request to its SystemGeolocator") {
-                guard case let (disposeBag?, systemGeolocator?) = unwrap(disposeBag, systemGeolocator) else { return }
+            condition("always") {
                 let address = "Valencia Airport"
-                sut?.geolocate(address: address).subscribe().disposed(by: disposeBag)
-                systemGeolocator.verifyCall(
-                    withIdentifier: FakeSystemGeolocator.Identifiers.geocodeAddressString, arguments: [address]
-                )
+                setup {
+                    sut?.geolocate(address: address).subscribe().disposed(by: disposeBag!)
```

Comment:
I like the idea of a helper (like your `unwrap`) that actually allows to print something more when unwrap crashes. I agree that _simple_ crash says only that _Unwrapping optional nil failed_ and that's it.